src/Sentry.Unity.Editor/AndroidManifestConfiguration.cs

```diff
@@ -1,3 +1,4 @@
+/*

```

Comment:
accident?

Reply:
Nope, I just commented the whole file until we need it. It uses some old parts, I just see no reason to change it now.